tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.yaml

```diff
@@ -2,12 +2,6 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: >-
   This template creates EFS, FSX Lustre, FSX Ontap, FSX Open ZFS storage that can be attached to a cluster.
 Parameters:
-  SubnetId0:
-    Type: String
-    Default: ''
-  SubnetId1:
-    Type: String
-    Default: ''
```

Comment:
CloudFormation parameters are removed because the subnet information is being handled by Jinja